src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -52,6 +52,10 @@ public boolean isShowTable() {
         return false;
     }
 
+    public boolean isShowBarChart() {
+        return false;
+    }
+
```

Comment:
Might need modified descriptions for this, since both functions just return false without any check carried out, it is unlikely that these functions will return true.

Reply:
This method is overridden by GenderBarChartCommandResult, SecLevelBarChartCommandResult, and SubjectBarChartCommandResult. It will return true in these 3 classes, if not any of these 3 classes calling this method, it can just return false because the BarChart Window should not be shown in that case.

Reply:
Added javadoc for this method.